addons/bus/tests/test_websocket_rate_limiting.py

```diff
@@ -0,0 +1,62 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+import time
+from websocket._exceptions import WebSocketProtocolException
```

Comment:
Is the `websocket` package required, just for the tests?

Reply:
@KangOl -- Yes, it was already used in [common.py](https://github.com/odoo-dev/odoo/blob/37bb2df07a11c455095a8cc2929ce186657cf43d/odoo/tests/common.py#L64.). This import miss the try/except statement and causes issues. I already created a PR to solve the problem: https://github.com/odoo/odoo/pull/98763/commits/8fe92eb21a896241d82ff7dd58da0280a586987b. If the websocket package is missing, the tests will be skipped: [bus/common.py](https://github.com/odoo-dev/odoo/blob/37bb2df07a11c455095a8cc2929ce186657cf43d/addons/bus/tests/common.py#L22)